lib/util.js

```diff
@@ -563,6 +569,8 @@ const addCommonScriptOptions = (command) => {
       'Path to the Chromium based executable to use to generate the CRX file')
     .option('-p, --publisher-proof-key <file>',
       'File containing private key for generating publisher proof')
+    .option('-a, --publisher-proof-key-alt <file>',
+      'Second private key for generating publisher proof. Useful when rotating keys.')
```

Comment:
a nit: look like a little inconsistency with the previous description: `File containing private key` vs `private key`.
Let's change one of them.

Reply:
Resolved in eaad711bf981606976fd3abaa5825cb123fd9f35.